### PR TITLE
DDCE-1171: updated reactive mongo dependencies and others

### PR DESCRIPTION
**Pull request by fraser-sprott**

### CODE REVIEW

File: project/MicroServiceBuild.scala [hunk 1]
```diff
@@ -13,23 +13,22 @@ private object AppDependencies {
 
   private val microserviceBootstrapVersion = "10.9.0"
   private val domainVersion = "5.3.0"
-  private val playReactivemongoVersion = "6.2.0"
-  private val scalatestPlusPlayVersion = "3.0.1"
+  private val playReactivemongoVersion = "6.8.0"
+  private val scalatestPlusPlayVersion = "1.5.1"
```
Comment (DannyPennington97) [inline project/MicroServiceBuild.scala:17]: Is there a reason we need to use the lower version here?

Comment (fraser-sprott) [reply project/MicroServiceBuild.scala:17]: Not sure what happened to that one

File: project/MicroServiceBuild.scala [hunk 2]
```diff
@@ -68,7 +67,7 @@ private object AppDependencies {
       override lazy val test = Seq(
         "uk.gov.hmrc" %% "reactivemongo-test" % reactivemongoTestVersion % scope,
         "org.scalatest" %% "scalatest" % scalatestVersion % scope,
-        "org.scalatestplus.play" %% "scalatestplus-play" % "1.5.1" % scope,
```
Comment (fraser-sprott) [inline project/MicroServiceBuild.scala:71]: this was why I changed it to 1.5.1, the variable wasn't being used, and the next version requires play 2.6

Comment (DannyPennington97) [reply project/MicroServiceBuild.scala:71]: Cool cool, looks good then

**Review by DannyPennington97 — APPROVED** [review]

**Pull request merged**